libraries/Aws/ServiceHealth/service_health.py

```diff
@@ -0,0 +1,91 @@
+from itertools import chain, product
+from datetime import datetime
+import requests
+import yaml
+from RW.Utils import parse_timedelta, csv_to_list
+
+GLOBAL_SERVICES = [  #List from https://d3s31nlw3sm5l8.cloudfront.net/services.json
+    "supportcenter",
+    "marketplace",
+    "chime",
+    "resourcegroups",
+    "organizations",
+    "iam",
+    "import-export",
+    "interregionvpcpeering",
+    "trustedadvisor",
+    "route53domainregistration",
+    "spencer",
+    "account",
+    "management-console",
+    "billingconsole",
+    "route53apprecoverycontroller",
+    "cloudfront",
+    "globalaccelerator",
+    "awswaf",
+    "health",
+    "apipricing",
+    "route53",
+    "chatbot",
+]
+
+
+class ServiceHealth:
+    def get_event_json(self, event_json_location: str) -> dict:
+        url = "https://history-events-{}-prod.s3.amazonaws.com/historyevents.json".format(event_json_location)
+
+        headers = {
+            "Accept": "application/json, text/plain, */*",
+            "Accept-Language": "en-US,en;q=0.9",
+            "Connection": "keep-alive",
+            "Origin": "https://health.aws.amazon.com",
+            "Referer": "https://health.aws.amazon.com/",
+        }
+        response = requests.request("GET", url, headers=headers, timeout=10)
+        if response.status_code != 200:
+            raise ValueError("Incident json file not found")
+
+        return yaml.safe_load(response.text)
```

Comment:
given the func name wouldn't we want `json.loads(response.json())` instead?

Reply:
Actually, from a purely technical point of view, loading the JSON with `json.loads(response.json())` and returning it, would still result in a python data structure, represented as YAML.

Provided that the main goal was to change the function's code in order to match its name (which is not), it should just return the response body, which is actually a JSON text, somewhere later in code.

And since JSON is a subset of YAML, I don't see anything wrong with parsing JSON with YAML parser.

So, in order to avoid confusion I just changed the method name, but I'm still open to discussion

Reply:
That works for me 😄